src/components/dev-hub/layout.js

```diff
@@ -90,6 +90,11 @@ export default ({ children, includeCanonical = true }) => {
             <GlobalWrapper>
                 <Helmet htmlAttributes={{ lang: 'en' }}>
                     <meta name="robots" content="index" />
+                    <link
+                        type="application/opensearchdescription+xml"
+                        rel="search"
+                        href="https://developer.mongodb.com/opensearch.xml"
```

Comment:
Ah looking at https://developer.mozilla.org/en-US/docs/Web/OpenSearch

we may need a title that matches the `ShortName` in the XML file